frontend/src/pages/AuthPage.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import {Auth} from "../components/Auth";
+import './AuthPage.css';
+
+export function AuthPage() {
+    return (
+        <>
+            <div className="title">
```

Comment:
Is this the correct class name?

Reply:
Yes. It is needed for the css-file that is only used in the AuthPage. So there should be no collision.